src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexInterpreter.cs

```diff
@@ -13,32 +12,25 @@ namespace System.Text.RegularExpressions
 {
     internal sealed class RegexInterpreter : RegexRunner
     {
-        internal int _runoperator;
-        internal int[] _runcodes;
-        internal int _runcodepos;
-        internal String[] _runstrings;
-        internal RegexCode _runcode;
-        internal RegexPrefix _runfcPrefix;
-        internal RegexBoyerMoore _runbmPrefix;
-        internal int _runanchors;
-        internal bool _runrtl;
-        internal bool _runci;
-        internal CultureInfo _runculture;
+        private readonly RegexCode _code;
+        private readonly CultureInfo _culture;
+        private int _operator;
+        private int _codepos;
+        private bool _rightToLeft;
+        private bool _caseInsensitive;
```

Comment:
These are all only used inside `RegexInterpreter` (even in the desktop code) so they can be `private`.